test/test_data/output_formatter.medium.fortio-noquirks.gold

```diff
@@ -0,0 +1,483 @@
+    "Count": 5
+   },
+   {
+    "Start": 47305,
+    "End": 47311,
+    "Percent": 40,
+    "Count": 7
+   },
+   {
+    "Start": 47311,
+    "End": 47317,
+    "Percent": 50,
+    "Count": 7
+   },
+   {
+    "Start": 47317,
+    "End": 47317,
+    "Percent": 55.000000000000007,
+    "Count": 0
+   },
+   {
+    "Start": 47317,
+    "End": 47321,
+    "Percent": 60,
+    "Count": 4
+   },
+   {
+    "Start": 47321,
+    "End": 47323,
+    "Percent": 65,
+    "Count": 2
+   },
+   {
+    "Start": 47323,
+    "End": 47327,
+    "Percent": 70,
+    "Count": 5
+   },
+   {
+    "Start": 47327,
+    "End": 47327,
+    "Percent": 75,
+    "Count": 0
+   },
+   {
+    "Start": 47327,
+    "End": 47329,
+    "Percent": 77.5,
+    "Count": 2
+   },
+   {
+    "Start": 47329,
+    "End": 47331,
+    "Percent": 80,
+    "Count": 3
+   },
+   {
+    "Start": 47331,
+    "End": 47331,
+    "Percent": 82.5,
+    "Count": 0
+   },
+   {
+    "Start": 47331,
+    "End": 47333,
+    "Percent": 85,
+    "Count": 2
+   },
+   {
+    "Start": 47333,
+    "End": 47333,
+    "Percent": 87.5,
+    "Count": 0
+   },
+   {
+    "Start": 47333,
+    "End": 47335,
+    "Percent": 88.75,
+    "Count": 1
+   },
+   {
+    "Start": 47335,
+    "End": 47339,
+    "Percent": 90,
+    "Count": 1
+   },
+   {
+    "Start": 47339,
+    "End": 47341,
+    "Percent": 91.25,
+    "Count": 1
+   },
+   {
+    "Start": 47341,
+    "End": 47341,
+    "Percent": 92.5,
+    "Count": 0
+   },
+   {
+    "Start": 47341,
+    "End": 47347,
+    "Percent": 93.75,
+    "Count": 1
+   },
+   {
+    "Start": 47347,
+    "End": 47347,
+    "Percent": 94.375,
+    "Count": 0
+   },
+   {
+    "Start": 47347,
+    "End": 47353,
+    "Percent": 95,
+    "Count": 2
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 95.625,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 96.25,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 96.875,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 97.1875,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 97.5,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 97.8125,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47353,
+    "Percent": 98.125,
+    "Count": 0
+   },
+   {
+    "Start": 47353,
+    "End": 47357,
+    "Percent": 98.4375,
+    "Count": 1
+   },
+   {
+    "Start": 47357,
+    "End": 47357,
+    "Percent": 100,
+    "Count": 0
+   }
+  ],
+  "Min": 47257,
+  "Max": 47357,
+  "Sum": 2649612,
+  "Avg": 47314.5,
+  "StdDev": 20.799210149838451,
+  "Percentiles": [
+   {
+    "Percentile": 50,
+    "Value": 47317
+   },
+   {
+    "Percentile": 75,
+    "Value": 47327
+   },
+   {
+    "Percentile": 80,
+    "Value": 47331
+   },
+   {
+    "Percentile": 90,
+    "Value": 47339
+   },
+   {
+    "Percentile": 95,
+    "Value": 47353
```

Comment:
are these value fake or real? as this is in seconds...

Reply:
I didn't come up with this test data, it's a copy of a pre-existing one modified to reflect the changes in here. If I interpret the file correctly, we're not looking at seconds, but rather a histogram of byte counts for the header?